docs/advanced_topics/api/v2/configuration.rst

```diff
@@ -127,7 +127,7 @@ For example:
     class BlogPage(Page):
         published_date = models.DateTimeField()
         body = RichTextField()
-        feed_image = models.ForeignKey('wagtailimages.Image', on_delete=models.SET_NULL, null=True, ...)
+        feed_image = models.ForeignKey('media.Image', on_delete=models.SET_NULL, null=True, ...)
```

Comment:
I would suggest leaving this as is

Reply:
Same as Andy, it seems to me like the docs should use the actual default. Perhaps it’d help if `wagtail start` provided feedback on what it did upon successful creation of the project, so people who use it are aware of there being a custom image model.